criu/kerndat.c

```diff
@@ -517,6 +551,25 @@ static bool kerndat_has_memfd_hugetlb(void)
 	return 0;
 }
 
+static bool kerndat_has_memfd_secret(void)
+{
+	int ret;
+
+	ret = memfd_secret(0);
+
+	if (ret > 0) {
+		kdat.has_memfd_secret = true;
+		close(ret);
+	} else if (ret == -1 && (errno == ENOSYS || errno == EINVAL || errno == EMFILE || errno == ENOMEM))
+		kdat.has_memfd_secret = false;
+	else {
```

Comment:
Either ALL branches of conditional should have {} or ALL branches should not have {}, there should not be any mixed thing.

Reply:
done, thanks!